CDAlertView/Classes/CDAlertView.swift

```diff
@@ -353,43 +355,48 @@ open class CDAlertView: UIView {
         }
     }
 
-    func keyboardWillShow(_ notification: Notification) {
-        if let userInfo = notification.userInfo {
-            if let keyboardSize = (userInfo[UIKeyboardFrameBeginUserInfoKey] as? NSValue)?.cgRectValue {
-                if let coverViewWindowCoordinates = coverView.superview?.convert(CGPoint(x: 0, y: coverView.frame.maxY), to: nil) {
-                    if coverViewWindowCoordinates.y > (keyboardSize.minY - keyboardSize.height) {
-                        popupCenterYPositionBeforeKeyboard = popupView.center.y
-                        let difference = coverViewWindowCoordinates.y - (keyboardSize.minY - keyboardSize.height)
-                        popupView.center.y -= difference
-                    }
-                }
-            }
+    @objc func keyboardWillShow(_ notification: Notification) {
+        guard let userInfo = notification.userInfo else {
+            return
         }
+        guard let keyboardSize = (userInfo[UIKeyboardFrameBeginUserInfoKey] as? NSValue)?.cgRectValue else {
+            return
+        }
+        guard let coverViewWindowCoordinates = coverView.superview?.convert(CGPoint(x: 0, y: coverView.frame.maxY), to: nil) else {
+            return
+        }
+        if (coverViewWindowCoordinates.y > (keyboardSize.minY - keyboardSize.height)) == false {
```

Comment:
it should be possible to write 
```
if coverViewWindowCoordinates.y <= (keyboardSize.minY - keyboardSize.height) {
  return 
}
```

to increase readability

Reply:
That's right. I'll fix.